examples/simple/wsgi_flask_app.py

```diff
@@ -6,15 +6,16 @@
 from flask import Flask
 from mitmproxy.addons import wsgiapp
 
+
 app = Flask("proxapp")
 
 
 @app.route('/')
-def hello_world():
+def hello_world() -> str:
     return 'Hello World!'
 
 
-def load(l):
+def load(l) -> wsgiapp.WSGIApp:
```

Comment:
I think we should keep the generic types for our hooks as defined in mitmproxy-core. Otherwise the type information is different for each addon, which is basically useless information... @cortesi, @mhils?